app/models/transfer_request.rb

```diff
@@ -224,7 +224,7 @@ def perform_transfer_of_contents
     target_asset.aliquots << aliquots_for_transfer
   rescue ActiveRecord::RecordNotUnique => e
     # We'll specifically handle tag clashes here so that we can produce more informative messages
-    raise e unless /aliquot_tags_and_tag2s_are_unique_within_receptacle/.match?(e.message)
+    raise e unless /aliquot_tag_tag2_and_tag_depth_are_unique_within_receptacle/.match?(e.message)
```

Comment:
There's similar code elsewhere in SS that will need to be updated:
Such as:
https://github.com/sanger/sequencescape/blob/a7c43e591668745dd1283ce3fecf2343781a39e4/app/models/tag_substitution.rb#L109

Reply:
Can't actually find other references at the moment, could have sworn we had a custom exception matcher on `Aliquot::TagClash` but know we also had a few bugs with handling that exception, so maybe it got ripped out / never got merged.